modules/ROOT/pages/prereq-software.adoc

```diff
@@ -8,12 +8,14 @@ The following sections list the software required to install and run the Anypoin
 == Supported Linux Distributions
 The following Linux distributions are supported:
 
-* Red Hat Enterprise Linux (RHEL), versions 7.4.x, and 7.5.x
+* Red Hat Enterprise Linux (RHEL), versions 7.4.x, 7.5.x, and 7.6.x
 
 * CentOS, versions 7.4.x, and 7.5.x
```

Comment:
`versions 7.4.x, 7.5.x, and 7.6.x` also applies here :)

Reply:
Added - thank you!